packages/block-library/src/navigation/edit/index.js

```diff
@@ -690,6 +696,51 @@ function Navigation( {
 		/>
 	);
 
+	const NAVIGATION_SIDEBAR_NAME = 'edit-site/block-inspector';
+	const {
+		isNavigationSidebarOpen,
+	} = useSelect(
+		( select ) => {
+			return {
+				isNavigationSidebarOpen:
+					select( interfaceStore ).getActiveComplementaryArea(
+						editSiteStore.name
```

Comment:
@scruffian If it helps we could consider whether it's worth importing the entire store just to access its `name` property. You should just hard code the value and have done.

Reply:
That's a good point. Maybe we can extract the name to a const